src/shared/infra/database/typeorm/migrations/1614729383843-Crime.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from 'typeorm';
+
+export class Crime1614729383843 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'tb_crime',
+        columns: [
+          {
+            name: 'id',
+            type: 'uuid',
+            isPrimary: true,
+            generationStrategy: 'uuid',
+            default: 'uuid_generate_v4()',
+          },
+          {
+            name: 'date',
+            type: 'date',
+            isNullable: true,
```

Comment:
This argument cannot be null, you must have confused it with the time attribute, as it can be null